.github/workflows/binary.yaml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   binary:
-    name: Create
+    name: Create the dune binary
```

Comment:
I think this actually creates a tarball now. Which is indeed "binary" but not in the sense that most people mean it. Maybe better call it "artifact" (along with the whole job).

Reply:
Good call
I just didn't like the orphaned 'Create' name